docs/embedded/concepts/spe-copilot.md

```diff
@@ -0,0 +1,262 @@
+---
+title: SharePoint Embedded Copilot
+description: Details Copilot Integration with SharePoint Embedded Applications
+ms.date: 02/19/2025
+ms.localizationpriority: high
+---
+
+# Overview
+
+> [!NOTE]
+>
+> Sharepoint Embedded Copilot is currently in public preview, the API surface and SDK are expected to change frequently.
+
+SharePoint Embedded Copilot is a powerful tool designed to enhance the functionality of SharePoint Embedded applications by integrating advanced Microsoft 365 features ( Purview, Protection etc. )
+
+![Diagram illustrating SPE copilot is AI ready](../images/speco-apparch.png)
+
+Currently, the only way to incorporate this feature into your custom application is through our React SDK library written in TypeScript. Plans to support future runtimes will be announced. The SDK is configured with the containerid instance of your containertype, as well as the authorization and authentication token logic you provdide through a callback. It will embed itself as an iframe into your host application. By default, the iframe is given a frame-ancestors property that prevents it from being embedded by any host until configured. Details are provided below.
+
+## Key Features
+
+![Showcasing how SPE content gets AI from M365 copilot](../images/speco-aivalue.png)
+
+### BIlling/Licensing
+
+Currently to use SPE copiot the consuming tenant user of the application is required to have a copiot license. In the future the license based model will be deprecating and the copilot usage will be charged on a pay-as-you-go basis to the owner of the owning tenant of the container type.
+
+### Application Scoping
+
+Application scoping in SharePoint Embedded Copilot (SPE Copilot) involves defining the boundaries and context within which the tool operates, ensuring its features and capabilities are tailored to meet the specific needs of different applications. This process helps customize the Copilot's functionality, making it more effective and relevant for various use cases.
+
+When SPE Copilot users query the LLM, it will only have access to files that the `User+Application` have access to, which would mean a minima of a superset. Typically this will be the application that has been granted access over the container type which will typically be more data then just what the user has access to.

```

Comment:
`minimal` (typo)

That said, we normally use terms like "intersection" and "effective permissions" for this.

> The effective permissions for the Declarative Agent session will be the intersection of your SharePoint Embedded application's permissions and the user's permissions.